standard/wallets/history.mdx

```diff
@@ -172,11 +182,11 @@ Essentially, `subwallet_id` is just a number added to the contract state when it
 
 | Exit code | Description                                                             |
 |-----------|-------------------------------------------------------------------------|
-| 0x23      | `valid_until` check failed; transaction confirmation attempted too late |
-| 0x24      | `Ed25519 signature` check failed                                        |
-| 0x21      | `seqno` check failed; replay protection triggered                       |
-| 0x22      | `subwallet_id` does not match the stored one                            |
-| 0x0       | Standard successful execution exit code.                                |
+| 33        | `seqno` check failed; replay protection triggered                       |
+| 34        | `subwallet_id` does not match the stored one                            |
+| 35        | `valid_until` check failed; transaction confirmation attempted too late |
+| 35        | `Ed25519 signature` check failed                                        |
```

Comment:
We should note somewhere the `35` exit code is re-used, so people (and hopefully) AI understand it is not a type